index.html

```diff
@@ -131,197 +131,18 @@ <h3 class="connecting">LET'S CONNECT</h3>
-                                    </li>
-                                </ul>
-                                <p class="paragraph">
-                                    A daily selection of privately <br> personalized reads; no accounts or <br> sign-ups required.
-                                </p>
-                                <ul class="language-button">
-                                    <li><button type="button" class="l-button">html</button></li>
-                                    <li><button type="button" class="l-button">css</button></li>
-                                    <li><button type="button" class="l-button">javaScript</button></li>
-                                </ul>
-                                <button type="button" class="see-project">See Project</button>
-                            </div>                    
-                        </div>
-                        <!-- Desktop only version of works card -->
-                        <div class="card card-desk">
-                            <div class="col2">
-                                <h2 class="main-heading">
-                                    Multi-Post Stories
-                                </h2>
-                                <ul class="top-container">
-                                    <li>
-                                        <p class="canopy">CANOPY</p>
-                                    </li>
-                                    <li><img src="./pics/dot.png" alt="dot"></li>
-                                    <li>
-                                        <p class="dev">Front End Dev</p>
-                                    </li>
-                                    <li><img src="./pics/dot.png" alt="dot"></li>
-                                    <li>
-                                        <p class="dev year">2021</p>
-                                    </li>
-                                </ul>
-                                <p class="paragraph">
-                                    A daily selection of privately <br> personalized reads; no accounts or <br> sign-ups required.
-                                </p>
-                                <ul class="language-button">
-                                    <li><button type="button" class="l-button">html</button></li>
-                                    <li><button type="button" class="l-button">css</button></li>
-                                    <li><button type="button" class="l-button">javaScript</button></li>
-                                </ul>
-                                <button type="button" class="see-project">See Project</button>
-                            </div>
-                            <img class="screenshot" src="./pics/homepage4.png" alt="work4">
-                        </div>               
-                    </div>    
+                <!--Popup-->
+                <section class="detail-section">
```

Comment:
- [ ] In desktop, the popup does not follow the Figma design closely:

![image](https://user-images.githubusercontent.com/57421823/156441958-42229b63-5a33-47ce-bd5e-9b6680edaee2.png)

![image](https://user-images.githubusercontent.com/57421823/156441989-337b7d3e-07c3-4cf2-ae2f-bfdb04001d79.png)

Figma:

![image](https://user-images.githubusercontent.com/57421823/156442031-d4adff60-33a7-448d-a011-545d48cf1ed7.png)

Please, closely follow the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) given design implementing margins, padding, font styles, and Flexbox where is proper.

Reply:
I have tried to closely replicate it. it is not perfect but very close to it. Thanks